src/urllib3/contrib/_appengine_environ.py

```diff
@@ -17,12 +17,12 @@ def is_appengine_sandbox():
 
 def is_local_appengine():
     return ('APPENGINE_RUNTIME' in os.environ and
-            'Development/' in os.environ['SERVER_SOFTWARE'])
+            os.environ.get('SERVER_SOFTWARE', '').startswith('Development'))
 
 
 def is_prod_appengine():
     return ('APPENGINE_RUNTIME' in os.environ and
-            'Google App Engine/' in os.environ['SERVER_SOFTWARE'] and
+            not os.environ.get('SERVER_SOFTWARE', '').startswith('Development') and
```

Comment:
This is a different behavior than the one @theacodes had originally so I'll have her review it. :)

Reply:
I would say let's just `not is_local_appengine()`?